### PR TITLE
Fix target gene type plugin

### PR DESCRIPTION
**Pull request by josenavas**
This depends on https://github.com/biocore/qiita/pull/1796 so review/merge that one first.

### CODE REVIEW

**Comment by josenavas** [comment]
Ready for review!

**Comment by antgonza** [comment]
👍 @ElDeveloper, could you?

File: qiita_plugins/target_gene_type/target_gene_type/tests/test_summary.py [hunk 1]
```diff
@@ -139,47 +122,49 @@ def test_generate_html_summary(self):
...
+        self.assertTrue(obs_success)
+        self.assertIsNone(obs_ainfo)
+        self.assertEqual(obs_error, "")
 
         # asserting content of html
-        self.assertItemsEqual(html, EXP_HTML_DEMUX)
+        html_fp = join(self.out_dir, "artifact_%d.html" % self.artifact_id)
+        with open(html_fp) as html_f:
+            html = html_f.read()
+        print html
```
Comment (ElDeveloper) [inline qiita_plugins/target_gene_type/target_gene_type/tests/test_summary.py]: Can you remove these three prints?

**Comment by ElDeveloper** [comment]
👍

File: qiita_plugins/target_gene_type/target_gene_type/tests/test_summary.py (hunk 1 above)
Comment (josenavas) [inline qiita_plugins/target_gene_type/target_gene_type/tests/test_summary.py]: Whoops - Done!

**Pull request merged**